components/inputs/input-date.js

```diff
@@ -158,17 +161,24 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 		this.addEventListener('d2l-localize-behavior-language-changed', () => {
 			this._dateTimeDescriptor = getDateTimeDescriptorShared(true);
 			this.requestUpdate();
+			this._getContentWidth();
 		});
+
+		this._formattedValue = this.emptyStateText ? this.emptyStateText : '';
+		this._getContentWidth();
 	}
 
 	render() {
 		return html`
 			<d2l-dropdown ?disabled="${this.disabled}" no-auto-open>
 				<d2l-input-text
+					@blur="${this._handleInputTextBlur}"
 					@change="${this._handleChange}"
 					class="d2l-dropdown-opener"
+					content-width="${ifDefined(this._contentWidth)}"
```

Comment:
I'm not a huge fan of having to expose `content-width` on the text input... is there a way we could just add the width of the slot to `_contentWidth` and then use `<d2l-input-text style="max-width: X">`?

Reply:
That sounds promising - I'll give it a try